### DISCUSSION TITLE
How to disable deployment success notifications?

### DISCUSSION THREAD

**Discussion by iliubinskii** [vercel/community#2728] [Help] [answered]
### Summary

You can disable failure notifications:
![image](https://github.com/vercel/community/assets/70687432/99b1b988-fdbc-4cc6-a61d-518e998aa8ea)
but there is no way to disable success notifications.

Can this be added?
Successful deployment is an expected result that does not require any additional actions.
This is the first candidate for disabling notification about it.
And you don't even have a setting.

### Steps to Reproduce

Just open account settings:
https://vercel.com/account/notifications
and see that the option is missing.

**Comment by smaeda-ks** [comment]
@iliubinskii could you clarify which type of notification are you trying to disable? Email, Web, GitHub, Slack bot, etc

**Comment by iliubinskii** [comment]
Hi, thank you for getting back. I am trying to disable e-mail notifications for successful deployments (while continue to receive e-mail notifications for failed ones).

**Reply by amyegan** [reply]
Are the notifications coming from `notifications@vercel.com` or from a different address? I know sometimes people get notifications from GitHub about their deployments and that can be disabled by [configuring GitHub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).

**Comment by iliubinskii** [comment]
![image](https://github.com/vercel/community/assets/70687432/75a44a4d-be2e-4692-813f-fee9f1c28006)

**Comment by iliubinskii** [comment]
it looks to come from github, but the sender is vercel[bot].

**Comment by amyegan** [comment, marked as answer]
You can use [`github.silent`](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#github.silent) to stop the deployment bot commenting on pull requests and commits.

**Comment by iliubinskii** [comment]
Ok. Thank you!
